src/api/projects/queries/getProject/index.js

```diff
@@ -19,12 +22,49 @@ const resolver = async (
     throw new Error(AUTHENTICATION_ERROR_NO_USER)
   }
 
-  const project = await Project.query()
+  const project: Project = await Project.query()
     .eager(
-      '[locales, translationKeys.translationValues, translationKeys.translationValues.locale]'
+      '[translationKeys.translationValues, translationKeys.translationValues.locale]'
     )
     .findOne({ slug: projectSlug, ownerId: request.user.id })
 
+  project.locales = await Locale.query()
+    .join('projects_locales as pl', 'pl.localeId', 'locales.id')
+    .join('projects as p', 'pl.projectId', 'p.id')
+    .where('p.id', '=', project.id)
+    .orderBy('pl.id', 'ASC')
+
+  await project.locales.forEach(async locale => {
```

Comment:
For some reason I don't understand this loop doesn't set the values on the locale

Reply:
```js
await Promise.all(project.locales.map(async locale =>
  …
})
```

I guess that could work? I’m not sure